server.js

```diff
@@ -185,8 +185,25 @@ app.use('/patterns', require('./controllers/pattern-library'));
 /**
  * Handle Aliases
  */
+
+const removeWildcard = urlPath => urlPath.replace('/*', '');
+
 aliases.forEach(redirect => {
-    app.get(redirect.from, (req, res) => res.redirect(301, redirect.to));
+    // Allow wildcard redirects to avoid specifying a whole directory of links
+    // Currently we only support paths that end with a wildcard (eg. /foo/bar/*)
+    if (redirect.from.match(/\/\*$/)) {
+        const pathsMinusWildcards = {
```

Comment:
Could this be inlined `const wildcardPath = removeWildcard(redirect.to) + req.path.replace(removeWildcard(redirect.from), '');`